webui/react/.gitignore

```diff
@@ -18,5 +18,6 @@ build-storybook/
 .env.development.local
 .env.test.local
 .env.production.local
+node_modules/
```

Comment:
**question:** we have this defined in the `webui/.gitignore` which applies to `tests` as well. This is probably not needed?

Reply:
correct we don't need it for git uses. I recently realized some of the dev tools I use only go recursively down when looking at .gitignore files and not upwards. this as you mention is not needed for git itself but it would help other tools eg I'm using `fd` and `ag` silver searcher which look at .gitignore to limit what they return and I saw they don't go upwards.